src/styles/variables.css

```
@@ -0,0 +1,18 @@
+:root {
+  --main-font-color: #060b35;
+  --secondary-font-color: #616070;
+
+  --button-color: #00acdc;
```

```suggestion
  --button-blue: #00acdc;
```
It is good when variable are as much descriptive as they can, for better readability